### PR TITLE
Bump test to be compatible with polkadot-v0.9.22

### PR DESCRIPTION
**Pull request by seunlanlege**

### CODE REVIEW

File: modules/light-clients/11-beefy/types/update.go [hunk 1]
```diff
@@ -191,10 +191,7 @@ func (cs *ClientState) verifyHeader(
 	return nil
 }
 
-func (cs *ClientState) getMMRProf(beefyHeader *Header) (*mmr.Proof, error) {
-	// for debug purposes
-	var mmrLeavesRaw = make([]BeefyMmrLeaf, len(beefyHeader.ParachainHeaders))
-
+func (cs *ClientState) parachainHeadersToMmrProof(beefyHeader *Header) (*mmr.Proof, error) {
```
Comment (blasrodri) [inline modules/light-clients/11-beefy/types/update.go]: shouldn't it be parachainHeadersToMMRProof ?

File: modules/light-clients/11-beefy/types/update.go [hunk 2]
```diff
@@ -448,6 +321,8 @@ func (cs ClientState) UpdateState(ctx sdk.Context, cdc codec.BinaryCodec, client
 	for height, consensusState := range consensusStates {
 		// we store consensus state as (PARA_ID, HEIGHT) => ConsensusState
 		setConsensusState(clientStore, cdc, consensusState, height)
+
+		// TODO: pruning!
```
Comment (blasrodri) [inline modules/light-clients/11-beefy/types/update.go:325]: what's pruning in this context?

File: modules/light-clients/11-beefy/types/update_test.go [hunk 3]
```diff
@@ -5,8 +5,8 @@ import (
 	"encoding/binary"
 	"encoding/hex"
 	"fmt"
-	types2 "github.com/cosmos/cosmos-sdk/x/params/types"
-	"github.com/cosmos/ibc-go/v3/modules/core/02-client/keeper"
+	// types2 "github.com/cosmos/cosmos-sdk/x/params/types"
```
Comment (blasrodri) [inline modules/light-clients/11-beefy/types/update_test.go]: can't they be removed?

File: modules/light-clients/11-beefy/types/update_test.go [hunk 4]
```diff
@@ -68,10 +68,15 @@ func TestCheckHeaderAndUpdateState(t *testing.T) {
 	t.Log("====== subcribed! ======")
 	var clientState *types.ClientState
 	defer sub.Unsubscribe()
+	count := 1
 
 	for {
```
Comment (blasrodri) [inline modules/light-clients/11-beefy/types/update_test.go]: why not add the count condition in the loop directly?

File: modules/light-clients/11-beefy/types/update.go (hunk 1 above)
Comment (seunlanlege) [reply modules/light-clients/11-beefy/types/update.go]: Mmr is the naming convention.

File: modules/light-clients/11-beefy/types/update.go (hunk 2 above)
Comment (seunlanlege) [reply modules/light-clients/11-beefy/types/update.go:325]: pruning of old consensus states

File: modules/light-clients/11-beefy/types/update_test.go (hunk 4 above)
Comment (seunlanlege) [reply modules/light-clients/11-beefy/types/update_test.go]: my golang isn't that great tbh,

File: modules/light-clients/11-beefy/types/update_test.go [hunk 5]
```diff
@@ -354,16 +366,18 @@ func TestCheckHeaderAndUpdateState(t *testing.T) {
...
+			// //paramSpace = paramSpace.WithKeyTable(clientypes.ParamKeyTable())
 
-			k := keeper.NewKeeper(nil, nil, paramSpace, nil, nil)
-			ctx := sdk.Context{}
-			store := k.ClientStore(ctx, "1234")
+			// k := keeper.NewKeeper(nil, nil, paramSpace, nil, nil)
+			// ctx := sdk.Context{}
+			// store := k.ClientStore(ctx, "1234")
 
-			err = clientState.UpdateState(sdk.Context{}, nil, store, &header)
```
Comment (thomasmodeneis) [inline modules/light-clients/11-beefy/types/update_test.go:365]: is this supposed to be removed ?

Comment (seunlanlege) [reply modules/light-clients/11-beefy/types/update_test.go:365]: it was failing for me with nill derefrencing

File: modules/light-clients/11-beefy/types/update.go (hunk 1 above)
Comment (thomasmodeneis) [reply modules/light-clients/11-beefy/types/update.go]: he means that in Go, the convention is to capitalize acronyms, such as MMR

Comment (thomasmodeneis) [reply modules/light-clients/11-beefy/types/update.go]: I'll update the method name and resolve this

File: modules/light-clients/11-beefy/types/update_test.go (hunk 3 above)
Comment (thomasmodeneis) [reply modules/light-clients/11-beefy/types/update_test.go]: nop, they are needed for the updateState call further down. So I uncommented it and made the checks optional.

File: modules/light-clients/11-beefy/types/update_test.go (hunk 4 above)
Comment (thomasmodeneis) [reply modules/light-clients/11-beefy/types/update_test.go]: yes, good point. fixed.

File: modules/light-clients/11-beefy/types/update_test.go (hunk 5 above)
Comment (thomasmodeneis) [reply modules/light-clients/11-beefy/types/update_test.go:365]: cool, so I made this optional then.

**Pull request merged**